packages/runner/cypress/integration/issues/issue-8350.js

```diff
@@ -1,20 +1,21 @@
 const { createCypress } = require('../../support/helpers')
-const { runIsolatedCypress } = createCypress()
+const { verify } = createCypress()
 
+// https://github.com/cypress-io/cypress/issues/8214
+// https://github.com/cypress-io/cypress/issues/8288
 // https://github.com/cypress-io/cypress/issues/8350
-describe('issue-8350', () => {
-  it('does not hang on nested hook', () => {
-    runIsolatedCypress(() => {
-      before(() => {
-        beforeEach(() => {
-        })
-      })
+describe('issue-8350', { viewportHeight: 900 }, () => {
+  const file = 'nested_hooks_spec.js'
 
-      describe('s1', () => {
-        it('t1', () => {
-          //
-        })
-      })
-    })
+  verify.it('test', {
```

Comment:
Can we use a better test title that describes what it's testing? Like 'shows a proper error message when a beforeEach is nested in a before'?